### ISSUE TITLE
Daily Hacker News 29-06-2025

### ISSUE THREAD

**Issue by Foxhunt** [Foxhunt/daily-hackernews#353]
# Daily Hacker News
  
1. [**JavaScript Trademark Update**](https://deno.com/blog/deno-v-oracle4) `deno.com` [`comments`](https://news.ycombinator.com/item?id=44407139)  
2. [**Solving `Passport Application` with Haskell**](https://jameshaydon.github.io/passport/) `jameshaydon.github.io` [`comments`](https://news.ycombinator.com/item?id=44408872)  
3. [**MCP: An (Accidentally) Universal Plugin System**](https://worksonmymachine.substack.com/p/mcp-an-accidentally-universal-plugin) `worksonmymachine.substack.com` [`comments`](https://news.ycombinator.com/item?id=44404905)  
4. [**Addictions Are Being Engineered**](https://masonyarbrough.substack.com/p/engineered-addictions) `masonyarbrough.substack.com` [`comments`](https://news.ycombinator.com/item?id=44405057)  
5. [**Improving River Simulation**](https://undiscoveredworlds.blogspot.com/2025/04/improving-river-simulation.html) `undiscoveredworlds.blogspot.com` [`comments`](https://news.ycombinator.com/item?id=44382419)  
6. [**The Death of the Middle-Class Musician**](https://thewalrus.ca/the-death-of-the-middle-class-musician/) `thewalrus.ca` [`comments`](https://news.ycombinator.com/item?id=44408552)  
7. [**Life of an inference request (vLLM V1): How LLMs are served efficiently at scale**](https://www.ubicloud.com/blog/life-of-an-inference-request-vllm-v1) `www.ubicloud.com` [`comments`](https://news.ycombinator.com/item?id=44407058)  
8. [**We ran a Unix-like OS Xv6 on our home-built CPU with a home-built C compiler (2020)**](https://fuel.edby.coffee/posts/how-we-ported-xv6-os-to-a-home-built-cpu-with-a-home-built-c-compiler/) `fuel.edby.coffee` [`comments`](https://news.ycombinator.com/item?id=44404164)  
9. [**BusyBeaver(6) Is Quite Large**](https://scottaaronson.blog/?p=8972) `scottaaronson.blog` [`comments`](https://news.ycombinator.com/item?id=44406171)  
10. [**Community Is Motivation on Tap**](https://alanwu.xyz/posts/community/) `alanwu.xyz` [`comments`](https://news.ycombinator.com/item?id=44373562)  
11. [**Blackwell: Nvidia's GPU**](https://chipsandcheese.com/p/blackwell-nvidias-massive-gpu) `chipsandcheese.com` [`comments`](https://news.ycombinator.com/item?id=44409391)  
12. [**2025 ARRL Field Day**](https://www.arrl.org/field-day) `www.arrl.org` [`comments`](https://news.ycombinator.com/item?id=44407245)  
13. [**Group of investors represented by YouTuber Perifractic buys Commodore**](https://www.amiga-news.de/en/news/AN-2025-06-00123-EN.html) `www.amiga-news.de` [`comments`](https://news.ycombinator.com/item?id=44408459)  
14. [**Universal pre-training by iterated random computation**](https://arxiv.org/abs/2506.20057) `arxiv.org` [`comments`](https://news.ycombinator.com/item?id=44409555)  
15. [**Show HN: I'm an airline pilot – I built interactive graphs/globes of my flights**](https://jameshard.ing/pilot) `jameshard.ing` [`comments`](https://news.ycombinator.com/item?id=44396518)  
16. [**An Indoor Beehive in My Bedroom Wall**](https://www.keepingbackyardbees.com/an-indoor-beehive-zbwz1810zsau/) `www.keepingbackyardbees.com` [`comments`](https://news.ycombinator.com/item?id=44406178)  
17. [**Show HN: SVG Lined Tile Generator**](https://adpreese.github.io/svg-lined-tiles/) `adpreese.github.io` [`comments`](https://news.ycombinator.com/item?id=44383341)  
18. [**The Perils of 'Design Thinking'**](https://www.theatlantic.com/books/archive/2025/06/invention-of-design-maggie-gram-book-review/683302/) `www.theatlantic.com` [`comments`](https://news.ycombinator.com/item?id=44392104)  
19. [**Tennis Scorigami**](https://www.tennis-scorigami.com/) `www.tennis-scorigami.com` [`comments`](https://news.ycombinator.com/item?id=44388324)  
20. [**The European wood pigeon helped me appreciate its omnipresent city cousins**](https://www.nytimes.com/2025/06/24/magazine/pigeons-city-nature.html) `www.nytimes.com` [`comments`](https://news.ycombinator.com/item?id=44384693)  
21. [**Generative AI's crippling failure to induce robust models of the world**](https://garymarcus.substack.com/p/generative-ais-crippling-and-widespread) `garymarcus.substack.com` [`comments`](https://news.ycombinator.com/item?id=44409405)  
22. [**Is being bilingual good for your brain?**](https://www.economist.com/science-and-technology/2025/06/27/is-being-bilingual-good-for-your-brain) `www.economist.com` [`comments`](https://news.ycombinator.com/item?id=44406140)  
23. [**Refurb weekend: Gremlin Blasto arcade board**](http://oldvcr.blogspot.com/2025/06/refurb-weekend-gremlin-blasto-arcade.html) `oldvcr.blogspot.com` [`comments`](https://news.ycombinator.com/item?id=44409211)  
24. [**Gradient Descent Visualiser**](https://uclaacm.github.io/gradient-descent-visualiser/) `uclaacm.github.io` [`comments`](https://news.ycombinator.com/item?id=44378253)  
25. [**Show HN: AGL a toy language that compiles to Go**](https://github.com/alaingilbert/agl) `github.com` [`comments`](https://news.ycombinator.com/item?id=44378517)
